src/components/common/TransactionalFolioNumber.vue

```diff
@@ -62,8 +62,8 @@ export default class TransactionalFolioNumber extends Vue {
 
   /** Called when component is mounted. */
   mounted (): void {
-    // restore transactional FN if it exists, otherwise use account FN
-    this.localFolioNumber = this.transactionalFolioNumber || this.accountFolioNumber
```

Comment:
Is `accountFolioNumber` now obsolete? Please remove prop from here and parent components.